src/main/java/competition/subsystems/drive/commands/ArcadeDriveWithKinematicsCommand.java

```diff
@@ -0,0 +1,48 @@
+package competition.subsystems.drive.commands;
+
+import com.google.inject.Inject;
+
+import competition.operator_interface.OperatorInterface;
+import competition.subsystems.drive.DriveSubsystem;
+import xbot.common.command.BaseCommand;
+import xbot.common.math.MathUtils;
+
+public class ArcadeDriveWithKinematicsCommand extends BaseCommand {
+
+    final DriveSubsystem drive;
+    final OperatorInterface oi;
+
+    final double linearVelocityConstant = 0;
```

Comment:
if you make these `DoubleProperty`, then we can tune them while people are driving and find nice values. Also, zero is a pretty dangerous number, since it prevents the robot from driving at all.